File: LuckyBol/app/src/main/java/com/pplus/luckybol/core/network/service/CdnService.java
package com.pplus.luckybol.core.network.service;


import com.pplus.luckybol.core.network.apis.ICdnApi;

import com.pplus.networks.core.NetworkHeader;
import com.pplus.networks.service.AbstractService;
import okhttp3.OkHttpClient;
import retrofit2.Converter;

/**
 * Created by J2N on 16. 6. 20..
 */
public class CdnService extends AbstractService<ICdnApi> {

    @Override
    public String getEndPoint(){

        //map base url
        return "https://cdn.prnumber.com/";
    }

    @Override
    public OkHttpClient getCustomHttpClient(){

        return null;
    }

    @Override
    public Converter.Factory getCustomConverterFactory(){

        return null;
    }

    @Override
    public void initializeHeader(NetworkHeader header){

    }
}

File: PRNumber/app/src/main/java/com/pplus/prnumberuser/core/network/service/CdnService.kt
package com.pplus.prnumberuser.core.network.service

import com.pplus.networks.core.NetworkHeader
import com.pplus.networks.service.AbstractService
import com.pplus.prnumberuser.core.network.apis.ICdnApi
import okhttp3.OkHttpClient
import retrofit2.Converter

/**
 * Created by J2N on 16. 6. 20..
 */
class CdnService : AbstractService<ICdnApi?>() {
    override fun getEndPoint(): String {

        //map base url
        return "https://cdn.prnumber.com/"
    }

    override fun getCustomHttpClient(): OkHttpClient? {
        return null
    }

    override fun getCustomConverterFactory(): Converter.Factory? {
        return null
    }

    override fun initializeHeader(header: NetworkHeader) {}
}